unstructured/embed/interfaces.py

```diff
@@ -15,7 +15,6 @@ class EmbeddingConfig(BaseModel):
 class BaseEmbeddingEncoder(ABC):
     config: EmbeddingConfig
 
-    @abstractmethod
```

Comment:
Matches the implementation of `BaseEmbeddingEncoder` in `unstructured_ingest`: https://github.com/Unstructured-IO/unstructured-ingest/blob/main/unstructured_ingest/embed/interfaces.py#L16